go.mod

```diff
@@ -5,14 +5,15 @@ go 1.13
 require (
 	github.com/ONSdigital/dp-healthcheck v1.0.4
 	github.com/ONSdigital/dp-mongodb v1.2.0
-	github.com/ONSdigital/dp-net v1.0.3-0.20200519140208-ed50d1f9aa9a
+	github.com/ONSdigital/dp-net v1.0.4-0.20200521155446-34cf3d4fece1
```

Comment:
Is there a release version we can use for this?

Reply:
It is still in PR:
https://github.com/ONSdigital/dp-net/pull/6